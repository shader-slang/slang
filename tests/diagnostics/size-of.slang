// sizeof.slang

//DIAGNOSTIC_TEST:SIMPLE:

int func()
{
    return sizeof(func);
}
