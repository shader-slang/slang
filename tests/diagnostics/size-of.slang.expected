result code = -1
standard error = {
tests/diagnostics/size-of.slang(7): error 30099: argument to sizeof is invalid
    return sizeof(func);
           ^~~~~~
}
standard output = {
}
